docs/src/main/asciidoc/_javaee-guide/Jakarta_RESTful_Web_Services.adoc

```diff
@@ -9,14 +9,14 @@ ifdef::env-github[:imagesdir: ../images/]
 [[tutorial-overview]]
 == Tutorial Overview
 
-This chapter describes the Java API for RESTful web services (JAX-RS,
-defined in http://jcp.org/en/jsr/detail?id=311[JSR331]).
+This chapter describes the Jakarta RESTful Web Services
+(defined in http://jcp.org/en/jsr/detail?id=311[JSR331]).
 http://www.jboss.org/resteasy[RESTEasy] is an portable implementation of
 this specification which can run in any Servlet container. Tight
 integration with JBoss Application Server is available for optimal user
-experience in that environment. While JAX-RS is only a server-side
-specification, RESTeasy has innovated to bring JAX-RS to the client
-through the RESTEasy JAX-RS Client Framework.
+experience in that environment. While Jakarta RESTful Web Services is only a server-side
```

Comment:
Actually, "only a server-side" is out of date. Jakarta RESTful Web Services has a client framework now. We were there first, of course. ;-)